pallets/messages/src/tests.rs

```diff
@@ -419,12 +420,49 @@ fn get_messages_by_schema_with_ipfs_payload_location_should_return_offchain_payl
 		assert_eq!(
 			pagination_response.content[0],
 			MessageResponse {
-				msa_id: 10,
-				payload: cid,
+				payload: None,
+				index: from_index as u16,
+				provider_msa_id: 1,
+				block_number: 0,
+				payload_length: Some(IPFS_PAYLOAD_LENGTH),
```

Comment:
If I recall correctly, this was originally intended to be the size of the batch file referenced by the CID, not a constant, so that we could charge for off-chain messages based on file size.  In that case payload_length may not make sense as a name; it could be `message_size, and then it doesn't need to be None() for on-chain messages; in both cases it can be the length of the on-chain message or batch file.

Reply:
I could imagine `content_length` to follow the HTTP header, but I think it would need to be a different issue for re-naming or altering so that it can be discussed again.